wolox-books/src/app/services/user.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders } from '@angular/common/http';
+
+import { User } from '../screens/sign-up/interfaces/user';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+
+  readonly ROOT_URL = 'https://books-training-rails.herokuapp.com/api/v1';
```

Comment:
Use this like environment variable

Reply:
In variables inside classes please `cammelCase`, only use UPPER_CASE in constants defined outside classes or in external files.

Reply:
ok